### ISSUE TITLE
Avoid UserDefaults for Data Persistence

### ISSUE THREAD

**Issue by abeldemoz** [mattybwoy/Under-The-Weather#6]
User Defaults is supposed to be for small pieces of data such as settings/preferences. Weather data and cities should be persisted via Core Data or SwiftData

**Comment by mattybwoy** [comment]
Yeah I know, but I think its only storing cityIDs which are required to get weather and images. Full database seems overkill but will implement coredata for the next project :)
